freeglut/freeglut/src/fg_internal.h

```diff
@@ -572,9 +670,10 @@ do                                            \
 {                                             \
```

Comment:
not related to what you did, but any idea why there would be two definitions of INVOKE_WCB? Second is more compact but presumably first works on all platforms? I don't see a difference expect first is split up a bit more for legibility?